pulp_2to3_migration/app/pre_migration.py

```diff
@@ -535,10 +535,14 @@ def pre_migrate_distributor(repo_id, distributors, distributor_migrators, repo=N
                       'pulp2_type_id': dist_data.distributor_type_id,
                       'pulp2_last_updated': last_updated,
                       'pulp2_config': dist_data.config,
-                      'pulp2_repository': repo,
                       'pulp2_repo_id': repo_id,
                       'is_migrated': False})
 
+
+        if repo:
+            distributor.pulp2_repos.add(repo)
```

Comment:
You are adding a native repo here, is it on purpose?

Reply:
yes it is on purpose.
pulp2_repos will contain both native repos and repos from plan, this is needed to cover:
1. simple plan --> it will always take and serialize native objects
2. complex plan--> even though this relation will contain a mix of native relations and those specified in plan, in the serializer we will display only those distributors that are migrated and are in plan

Reply:
distributor.pulp2_repos will contain only native repos
repository.pulp3_dists will contain dists specified in plan

Reply:
We agreed to add in this condition the pulp3_dist relations as well to cover the simple migration plan